src/app/components/app-builds/app-builds.component.ts

```diff
@@ -151,4 +157,12 @@ export class AppBuildsComponent implements OnInit, OnDestroy {
   gotoBuild(buildId: number) {
     this.router.navigate(['build', buildId]);
   }
+
+  showPreviousBuilds(e: MouseEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+    this.fetch();
+    this.show = this.show * 2;
```

Comment:
why do you multiply `this.show`? what is that value?
you just need to increase `this.offset` by 5. `this.limit` is always 5.